R/register.R

```diff
@@ -258,12 +260,33 @@ get_call_entries <- function(path) {
   close(con)
 
   start <- grep("/* .Call calls */", res, fixed = TRUE)
+
   end <- grep("};", res, fixed = TRUE)
 
   if (length(start) == 0) {
     return("")
   }
-  res[seq(start, end)]
+
+  funs$package <- package
+
+  redundant <- as.character(glue_collapse_data(funs, 'extern SEXP _{package}_{name}', sep = '|'))
```

Comment:
You shouldn't need `as.character()` here, glue returns a subclass of character so they should work everywhere without issue.